src/adv.py

```diff
@@ -49,3 +70,37 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+def find_room_and_options():
+    for i in room:
+        if player1.current_room == room[i].name:
+            print(f'Current description - {room[i].description}')
+            print('Which way would you like to go?')
+            print('Here are your options')
+            for x in ['n_to', 's_to', 'e_to', 'w_to']:
+                d = getattr(room[i], x, False)
+                button = x.split('_')[0]
+                if(d is not False):
+                    print(f'For {d.name} press [{button}], {d.name} description - {d.description}')
+            return room[i]
+
+def move_player(cr, move):
+    m = move + '_to'
+    d = getattr(cr, m)
+    player1.current_room = d.name
+    return player1
+        
+
+def game():
```

Comment:
Nice, looks really clean. I like how you broke it up into different helper functions.